test/e2e_node/jenkins/copy-e2e-image.sh

```diff
@@ -16,10 +16,15 @@
 
 # Usage: copy-e2e-image.sh <image-name> <from-project-name> <to-project-name>
 
+# See *.properties for list of images to copy,
+# typically from kubernetes-node-e2e-images
+
 set -e
 set -x
 
 echo "Copying image $1 from project $2 to project $3..."
```

Comment:
store the args into named variables so this script is readable with less context.